popgen/mcmcSelection.go

```diff
@@ -146,6 +166,16 @@ func MetropolisHastings(data AFS, muZero float64, sigmaZero float64, iterations
 	}
 }
 
+func findAllN(data AFS) []int {
```

Comment:
Very minor comment since this is not the part of the code holding us back, but findAllN and findMaxN could be returned as two variables from the same function so that you only need to go through "data AFS" once.